tooling/api/src/bundle.ts

```diff
@@ -2,20 +2,20 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
-import 'regenerator-runtime/runtime'
-import * as app from './app'
-import * as cli from './cli'
-import * as dialog from './dialog'
-import * as event from './event'
-import * as updater from './updater'
-import * as fs from './fs'
-import * as path from './path'
-import * as http from './http'
-import * as shell from './shell'
-import * as tauri from './tauri'
-import * as window from './window'
-import * as notification from './notification'
-import * as globalShortcut from './globalShortcut'
+import 'https://unpkg.com/regenerator-runtime@0.13.7/runtime.js'
```

Comment:
nope.

Reply:
This is a complete PoC, so for now I imported it like this.

If this idea is somehow interesting for the team, we can definitely use [Trex](https://github.com/crewdevio/Trex), Deno's package manager, in order to keep the old code as it was, and just map the dependency in Trex's config file 😉

Reply:
Sure, just as long as we are installing from a tagged repo I could theoretically live with it.
`trex install --pkg [user]/[repo or repo@tag/branch]/[path/to/file] [packageName]`

Reply:
Yup, for now I'm doing it using the `import_map.json` file, which can be used without having to install Trex, and it build properly 👍